execution/src/gas.rs

```diff
@@ -164,6 +164,9 @@ impl L1GasFee for EcotoneGasFee {
             l1_base_fee_scalar: Some(self.base_fee_scalar.saturating_to()),
             l1_blob_base_fee: Some(self.blob_base_fee.saturating_to()),
             l1_blob_base_fee_scalar: Some(self.blob_base_fee_scalar.saturating_to()),
+            // TODO: What are these?
```

Comment:
Currently every TODO has an issue, we can create one for this too.

Reply:
We had a very similar one in [transaction.rs](https://github.com/MovedNetwork/op-move/blob/cf972b1d5c69b82d464f9bf615d382d69ffe0270/execution/src/transaction.rs#L102), but I couldn't find an issue for it, actually.

Reply:
#327 for new TODO.

#253 is the issue corresponding to the `None` fields in transactions.rs.